PP-10542: Process balance.available Stripe notification

oswaldquek opened the pull request.

stephencdaly commented on src/main/java/uk/gov/pay/connector/gateway/stripe/StripeNotificationService.java:
```diff
@@ -154,6 +158,31 @@ private void processDisputeNotification(StripeNotification notification, String
+            StripeBalance stripeBalance = toStripeBalance(notification.getObject());
+            if (!stripeBalance.getAvailable().isEmpty()) {
+                StripeBalance.Available available = stripeBalance.getAvailable().iterator().next();
+                logger.info("Logging stripe balance",
```
Would it be worth adding in a comment about how we intend to use this?  I can see myself coming along and wondering that this is for and what will it affect if I want to change something about it in the future?

stephencdaly reviewed (commented):
Code changes look good.

It would be good to have a code comment and some detail in the commit message about how this will be used so our future selves know why this was added.

oswaldquek commented:
@stephencdaly I've [addressed your review comments which has more detail in the commit message](https://github.com/alphagov/pay-connector/pull/4232/commits/b1c3528c607c00682c389e3f5f6d61a84ec413a4). If you approve I will squash and merge the commits to make it one commit with the message.

stephencdaly reviewed (approved):
LGTM!
